Spanner/tests/EmulatorTestTrait.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Spanner\Tests;
+
+/**
+ * Provides checks for whether to skip tests when the emulator is being used
+ *
+ * @experimental
+ * @internal
+ */
+trait EmulatorTestTrait
```

Comment:
I'm sorry for not being clearer. We don't need a trait here. Add `checkAndSkipEmulatorTests()` to `Google\Cloud\Spanner\Tests\System\SpannerTestCase`.

Reply:
I'm sorry, apparently I was only viewing a subset of changes. Disregard this.